lib/appsignal/transaction/registry.ex

```diff
@@ -43,14 +43,16 @@ defmodule Appsignal.TransactionRegistry do
   """
   @spec lookup(pid, boolean) :: Transaction.t | nil
   def lookup(pid, return_removed \\ false) do
-    case :ets.lookup(@table, pid) do
-      [{^pid, :removed}] ->
-        case return_removed do
-          false -> nil
-          true -> :removed
-        end
-      [{^pid, transaction}] -> transaction
-      [] -> nil
+    if Appsignal.started? do
+      case :ets.lookup(@table, pid) do
+        [{^pid, :removed}] ->
+          case return_removed do
+            false -> nil
+            true -> :removed
+          end
+        [{^pid, transaction}] -> transaction
+        [] -> nil
+      end
```

Comment:
Adding an `else` clause would be more explicit here (to return `nil`).

Alternatively you can do a shortcircuit using `case Appsignal.started? && :ets.lookup(@table.pid) do` (with a `false ->` clause added) to keep the indentation on the same level.